### ISSUE TITLE
Fix code errors found in accessibility review

### ISSUE THREAD

**Issue by rianrietveld** [wpaccessibility/wp-a11y-docs#235]
From the accessibility audit, issue https://github.com/wpaccessibility/wp-a11y-docs/issues/232

Code errors or classes that are not correctly added or are missing while implementing the new design:
- This blogpost also has larger intro text, but without the light grey line: https://wpaccessibility.org/blog/2025/accessible-error-messages-forms/ .
- One blogpost has an introduction-text that is styled differently than the rest of the article. It has a very light grey line (#F6F6F7) next to it and uses a larger font. The light grey line marks that this is a different type of content from the rest of the article. This means it needs to have a contrast ratio of at least 3:1. Alternatively consider removing the line. https://wpaccessibility.org/blog/2025/offer-multiple-ways-of-contact/ 

The work:
Fix the issues.
